godtools/Constants/GTGlobalTractBindings.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  GTGlobalTractBindings.swift
+//  godtools
+//
+//  Created by Pablo Marti on 6/8/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import UIKit
+
+class GTGlobalTractBindings: NSObject {
+    
+    enum Bindings:String {
+        case followUp = "followup:send"
+    }
+    
+    static func listen(listener: String, element: BaseTractElement) {
+        switch listener {
+        case Bindings.followUp.rawValue:
+            let manager = FollowUpsManager()
+            _ = manager.createSubscriber(params: element.getFormData()!)
```

Comment:
`getFormData()` can return nil, I would extract out that call and use a `guard let` assignment to avoid any crashes here.